File: design-patterns/go/abstract_factory/transport/car.go
// car.go

package main

import "fmt"

type Car struct {
}

func NewCar() (car *Car) {
	car = &Car{}
	return
}

func (car *Car) Start() {
	fmt.Println("Car Started")
}

func (car *Car) Stop() {
	fmt.Println("Car Stopped")
}

func (car *Car) Repair() {
	fmt.Println("Car Repair")
}

File: design-patterns/php/factory/transport-method/Car.php
<?php
// Car.php

namespace Factory\TransportMethod;

class Car implements Transport {
    public function start() {
        echo "Car started\n";
    }

    public function stop() {
        echo "Car Stopped\n";
    }

    public function repair() {
        echo "Car Repair\n";
    }
}